apps/src/templates/sectionProgress/VirtualizedDetailView.jsx

```diff
@@ -121,6 +121,12 @@ export default class VirtualizedDetailView extends Component {
     const rowCount = section.students.length + 2;
     // Add 1 to account for the student name column
     const columnCount = scriptData.stages.length + 1;
+    const rowHeight = 40;
+    // Calculate height based on the number of rows
+    const tableHeightFromRowCount = rowHeight * rowCount;
+    // Use a 'maxHeight' of 680 for when there are many rows
+    const tableHeight = tableHeightFromRowCount < 680 ?
```

Comment:
`Math.max` should do this for you: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max

Reply:
I'd have to get confirmation from @islemaster but I think it's generally preferred to use built in functions for math like this because it's already tested and the functions are easy to read.

Reply:
neat! I like this way better.  Thanks for sharing!